packages/core/src/VRange.ts

```diff
@@ -95,6 +97,13 @@ export class VRange {
     // Public
     //--------------------------------------------------------------------------
 
+    get mode(): Mode {
+        if (!this._mode) {
+            this._mode = this.editor.plugins.get(Mode);
+        }
+        return this._mode;
+    }
+
```

Comment:
Pourquoi lui il a besoin d'un have avec `mode` et `_mode` alors que les autres ça marche très bien de faire `mode = this.editor.plugins.get(Mode)` et c'est tout ?